### ISSUE TITLE
Panic Error on merging

### ISSUE THREAD

**Issue by monksy** [parse-nl/SystemdJournal2Gelf#1]
I get the following error on a merge:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x402d76]

goroutine 1 [running]:
panic(0x602900, 0xc820010130)
        /usr/lib/go/src/runtime/panic.go:481 +0x3e6
main.main()

```

**Comment by SjonHortensius** [comment]
Can you find out what journal entry it fails on? If you execute `journalctl --all --output=json` you get the same output as SystemdJournal2Gelf processes

**Comment by SjonHortensius** [comment]
this would also happen if journalctl had issues with parameters, eg. `SystemdJournal2Gelf localhost:12201 x` would result in this. That has also been fixed in 8b37b7b31f32cd654d1d7bca01b09a5823de2611
